packages/api/src/index.js

```diff
@@ -109,11 +109,13 @@ function serverError (error, request, env) {
 
 export default {
   async fetch (request, env, ctx) {
+    let response
     try {
-      return await router.handle(request, env, ctx)
+      response = await router.handle(request, env, ctx)
     } catch (error) {
-      return serverError(error, request, env)
+      response = serverError(error, request, env)
     }
+    return env.log.end(response)
```

Comment:
As soon as I saw this the first thought I had was: why are we returning the logs here.
Obviously that's not the case, but still feels a bit un-natural `log.end` takes a response, mutates it and returns it.

Not a big deal and possibly not something for this PR, but I'd love to hear what you think?

Reply:
Yeah, I agree that it seems a bit odd. Are you thinking that it would make more sense to do:

```javascript
await env.log.end(response)
return response
```

(The `end` method does this: `response.headers.set('Server-Timing', this._timersString())` so we need to `await` it before we return the response.)

Reply:
Not sure what's best here, I guess my problem is mostly about `log.end` mutating the response itself.
 But to be honest, it is not a big deal.
So either what we have or what we're suggesting is ok. happy for you to go with what you prefer.

Reply:
Ok, cool. I've gone for the slight tweak, because I agree that previously it looked like we were returning the logs!